defaults/main.yml

```diff
@@ -4,8 +4,16 @@
 monasca_checks: {}
 monasca_agent_system_only: False
 agent_reconfigure_script: /usr/local/bin/monasca-reconfigure
+monasca_agent_check_frequency: 60
```

Comment:
I prefer to rely on the default check frequency built into the agent itself rather than setting one here.